### DISCUSSION TITLE
Basic  / makeshift warming filter

### DISCUSSION THREAD

**Discussion by Victor20x** [ImageMagick/ImageMagick#2939] [Help]
I'm using IM to do some very basic stuff as part of a longer Powershell script. Ideally I would apply a warming filter to the images I am working with at the same time. I have found Fred's Image Magick scripts but it looks like I'd have to install Cygwin to use those under Powershell, and I'm not keen on doing that. Is there a rough and ready way to get this kind of effect? There's a slightly blue cast to the images that I'd like to turn beige.

I'm just looking for a starting point I can investigate this from.

Thanks.

**Comment by fmw42** [comment]
Please identify your IM version and post an example image.  You could consider using -modulate to shift the colors towards the red.

**Comment by snibgo** [comment]
There are many ways to do this. One of the simplest it to multiply the red channel by some number, and divide the blue channel by the same number:
```
magick in.png -channel R -evaluate Multiply 1.05 -channel B -evaluate Divide 1.05 +channel out.png
```
